pkg/metrics/statsd/metrics.go

```diff
@@ -0,0 +1,116 @@
+package statsd
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"time"
+)
+
+// Generic represents generic metric structure
+type Generic struct {
+	Metric    string
+	Account   string
+	Namespace string
+	Dims      map[string]string
+	TS        time.Time
+}
+
+// Float point number is used to represent floating point numbers which vary a
+// lot in its magnitude and thus warrant storage in a mantissa + expontent format
+// (such as physical quantities, results of complex calculations, etc). <value>
+// can be any fractional number, which would be stored as double precision IEEE 754 float.
+type float struct {
```

Comment:
there's a lot of duplication in this file to say that the only meaningful difference I can see is `:%d|g` or `:%f|f`